app/controllers/transcriptions_controller.rb

```diff
@@ -24,6 +26,27 @@ def update_attrs
     jsonapi_deserialize(params)
   end
 
+  # jsonapi.rb filtering doesn't handle filtering by enum term (e.g. 'ready'),
+  # so we must translate status terms to their integer value if they're present
+  def status_filter_to_int
+    if params[:filter]
+      params[:filter].each do |key, value|
+        if key.split('_').first == 'status'
```

Comment:
minor, not blocking - it might be clearer to use a regex here as i wasn't sure why you were splitting the filter keys
```
# value filter keys might include *_eq, *_in (https://github.com/activerecord-hackery/ransack#search-matchers)
key =~ /\Astatus_.+\z/
```

Reply:
I'll put in a comment for clarity, as I want to merge this asap